plugins/modules/database/influxdb/influxdb_retention_policy.py

```diff
@@ -134,6 +153,21 @@
 DURATION_REGEX = re.compile(r'(\d+)(ns|u|µ|ms|s|m|h|d|w)')
 EXTENDED_DURATION_REGEX = re.compile(r'(?:(\d+)(ns|u|µ|ms|m|h|d|w)|(\d+(?:\.\d+)?)(s))')
 
+DURATION_UNIT_NANOSECS = {
+    'ns': 1,
+    'u': 1000,
+    'µ': 1000,
```

Comment:
I'm not sure about this, could adding `µ` possibly cause some issues later on, formatting-wise?

Reply:
On #2383 `µ` was already included, this is only a rework to simplify that part of code.

Just out of curiosity, what kind of issues could it cause? I think it's an standard character in all encodings (I've checked extended ASCII, UTF-8, Windows-1252) but it's true that no one in their right mind would use `µ` instead of `u`. I just included it to be fully compatible with InfluxDB spec.

Reply:
OK then, thanks for checking!